code/modules/antagonists/changeling/powers/gaseous_pores.dm

```diff
@@ -0,0 +1,39 @@
+/datum/action/changeling/gaseous_pores
+	name = "Gaseous Pores"
+	desc = "Our skins bursts, releasing somniferous gases to put opponents to sleep and cover our retreat."
+	helptext = "Our kind are immune to the gases internals are not necessary"
+	button_icon_state = "smoke"
+	chemical_cost = 35
+	dna_cost = 2
+	req_stat = UNCONSCIOUS
+	var/range = 4
```

Comment:
make this a define

Reply:
How would i go about this?

Reply:
You'd put something like `#define GAS_PORE_RANGE 4` at the top of the file and `#undef GAS_PORE_RANGE` at the bottom

Reply:
eh its only used once so a define is pointless